tap-snapshots/test/lib/config.js.test.cjs

```diff
@@ -5,6 +5,418 @@
  * Make sure to inspect the output below.  Do not ignore changes!
  */
 'use strict'
+exports[`test/lib/config.js TAP > must match snapshot 1`] = `
+Object {
+  "init-author-name": Definition {
+    "default": "",
+    "defaultDescription": "/"/"",
+    "description": String(
+      
+          The value \`npm init\` should use by default for the package author's name.
+        
+    ),
+    "envExport": true,
+    "hint": "<init-author-name>",
+    "key": "init-author-name",
+    "type": Function String(),
+    "typeDescription": "String",
+    "usage": "--init-author-name <init-author-name>",
+  },
+  "init-version": Definition {
+    "default": "1.0.0",
+    "defaultDescription": "/"1.0.0/"",
+    "description": String(
+      
+          The value that \`npm init\` should use by default for the package
+          version number, if not already set in package.json.
+        
+    ),
+    "envExport": true,
+    "hint": "<init-version>",
+    "key": "init-version",
+    "type": Object {
```

Comment:
whoa. what are we doing here? this looks like the entire contents of the real semver class and that seems wrong for usage output lol